fauna/functions/sort_members_by_created_at_asc.js

```diff
@@ -0,0 +1,34 @@
+import { query as q } from 'faunadb'
```

Comment:
We have the option to use named imports instead of prefixing all commands with `q`. Named imports make the FQL more readable but the import statement becomes more cumbersome to manage because we need to import each FQL function separately. In one of my projects I regretted using named imports when my UDFs had 25+ named imports 😅